### PR TITLE
[syncd] skip ledinit during syncd warm start

### PR DESCRIPTION
**Pull request by yxieca**
Signed-off-by: Ying Xie <ying.xie@microsoft.com>

**- What I did**
Skip ledinit during syncd warm restart

### CODE REVIEW

**Comment by lguohan** [comment]
what about syncd docker restart?

File: files/scripts/syncd.sh [hunk 1]
```diff
@@ -64,8 +64,13 @@ start() {
...
 
-    # Don't flush DB during warm boot
-    if [[ x"$WARM_BOOT" != x"true" ]]; then
+    if [[ x"$WARM_BOOT" == x"true" ]]; then
+        # Leave a mark for syncd scripts running inside docker.
+        touch /host/warmboot/warm-starting
+    else
+        rm -f /host/warmboot/warm-starting
+
+        # Don't flush DB during warm boot
```
Comment (zhenggen-xu) [inline files/scripts/syncd.sh]: Looks more straight if we change the comment to: "# Flush ASIC DB during cold restart/reboot"?

**Comment by yxieca** [comment]
retest this please

File: files/scripts/syncd.sh (hunk 1 above)
Comment (yxieca) [reply files/scripts/syncd.sh]: Copy & Paste sins. Sharp eyes!

**Comment by yxieca** [comment]
retest this please

**Pull request merged**